cpp/src/Glacier2/SessionRouterI.cpp

```diff
@@ -824,13 +824,15 @@ SessionRouterI::destroySession(const ConnectionPtr& connection)
 int64_t
 SessionRouterI::getSessionTimeout(const Ice::Current& current) const
 {
-    return current.con->getACM().timeout;
+    return getACMTimeout(current);
 }
 
 int
-SessionRouterI::getACMTimeout(const Ice::Current& current) const
+SessionRouterI::getACMTimeout(const Ice::Current&) const
```

Comment:
Wasn't the idea to remove these two methods and require to setup the same idle timeout on both sides?